bidi-scanner/github/package.json

```diff
@@ -2,9 +2,9 @@
   "name": "@edfi/bidi-scanner-action",
   "type": "module",
   "engines": {
-    "node": ">=16.0.0"
+    "node": ">=18.0.0"
```

Comment:
Node 18 _does_ exist as an option when running custom code. You just can't choose `node18` in `action.yml`.